tokio-console/README.md

```diff
@@ -97,30 +97,87 @@ tokio-console http://my.instrumented.application.local:6669
 
 See [here][cli-ref] for a complete list of all command-line arguments.
 
+Tokio Console has a numnber of different views:
+* [Tasks List](#tasks-list)
+* [Task Details](#task-details)
+* [Resources List](#resources-list)
+* [Resource Details](#resource-details)
+
+### Tasks List
+
 When the console CLI is launched, it displays a list of all [asynchronous tasks]
 in the program:
 
 ![tasks list](https://raw.githubusercontent.com/tokio-rs/console/main/assets/tokio-console-0.1.8/tasks_list.png)
 
+Tasks are displayed in a table.
+
+* `Warn` - The number of warnings active for the task.
+* `ID` - The ID of the task. This is the same as the value returned by the unstable [`tokio::task::Id`](https://docs.rs/tokio/latest/tokio/task/struct.Id.html) API (see documentation for details).
+* `State` - The state of the task.
+  * `RUNNING` ▶ - Task is currently being polled.
+  * `IDLE` ⏸ - Task is waiting on some resource.
+  * `SCHED` ⏫ - Task is scheduled (it has been woken but not yet polled).
+  * `DONE` ⏹ - Task has completed.
```

Comment:
Nit, take it or leave it: this kinda looks like it's saying we will display both the text *and* the emoji, but we only display one or the other. Maybe:
```suggestion
  * `RUNNING`/▶ - Task is currently being polled.
  * `IDLE`/⏸ - Task is waiting on some resource.
  * `SCHED`/⏫ - Task is scheduled (it has been woken but not yet polled).
  * `DONE`/⏹ - Task has completed.
```